### PR TITLE
Add Veepee as adopter of the solution

### PR DESCRIPTION
**Pull request by nerzhul**
Add [Veepee](https://www.veepee.com) as an adopter of the solution.

**Warning**, this is the stats for our biggest Kubernetes cluster at this moment. 
If you wants me to summary all our Kubernetes clusters (as they are part of a big federation) tell me.

### CODE REVIEW

**Comment by rockyluke** [comment]
Thanks @nerzhul for sharing 👍

File: ADOPTERS.md [hunk 1]
```diff
@@ -40,6 +40,19 @@ Details (optional):
...
+https://www.veepee.com
+
+Environments: Bare Metal
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Details (optional):
+- HA Pair of Prometheus
+- 108161 samples/s
+- 2.3M active series
```
Comment (lilic) [inline ADOPTERS.md:54]: Very cool!

**Review by lilic — APPROVED** [review]
Nice, thank you so much! 🎉

**Comment by paulfantom** [comment]
Resolved conflict as a result of a different PR. Merging this as CI already passed and doesn't really check anything for this file.

**Comment by brancz** [comment]
I'd love to see the total series and samples/s as well if you can share! (also across how many clusters)

Super cool! Thank you!

**Comment by nerzhul** [comment]
@brancz i publish the informations next sprint by a PR on this repository :)

**Pull request merged**